src/daemon/rpmostreed-os-experimental.cxx

```diff
@@ -247,52 +243,62 @@ osexperimental_handle_download_packages (RPMOSTreeOSExperimental *interface,
           hy_query_filter (query, HY_PKG_REPONAME, HY_EQ, source_name);
           pkglist = hy_query_run (query);
           if (!pkglist || pkglist->len == 0)
-            {
-              glnx_throw (&local_error, "No matches for \"%s\" in repo '%s'", pkg_name, source_name);
-              goto out;
-            }
+            return glnx_prefix_error (error, "No matches for \"%s\" in repo '%s'", pkg_name, source_name);
```

Comment:
Self note: this is wrong, should have been a `glnx_throw()` instead, see https://github.com/coreos/rpm-ostree/issues/3344.